sections/featured-blog.liquid

```diff
@@ -129,6 +129,22 @@
       "default": 3,
       "label": "t:sections.featured-blog.settings.columns_desktop.label"
     },
+    {
+      "type": "select",
+      "id": "columns_mobile",
+      "default": "2",
+      "label": "t:sections.featured-blog.settings.columns_mobile.label",
+      "options": [
+        {
+          "value": "1",
+          "label": "t:sections.featured-blog.settings.columns_mobile.options__1.label"
+        },
+        {
+          "value": "2",
```

Comment:
@katycobb @wiktoriaswiecicka @ludoboludo I am actually not sure if we should have this setting for mobile on blog. 🤔 Because of the text blurb you could end with something like this

https://screenshot.click/14-44-qtpbx-ep8zg.mp4

Reply:
Fair point 🤔  but since it impacts also tablets, I'm guessing we should have it there ? It's one of those special case where 2 columns would make sense on tablet but one on mobile.

Reply:
I agree that we can skip offering this "number of columns on mobile" setting on the blog posts section. Even when the blurb is small it looks squished, and so is the heading.